tasks/eva_3417/src/main/groovy/eva3417/RemediateLowerCaseRefAlt.groovy

```diff
@@ -0,0 +1,462 @@
+            rsSet.add(sveInDB.getClusteredVariantAccession())
+
+            if(rsSet.size() > 1){
+                //all sve involved does not have same RS, document SVEs in file - don't process
+                documentInFile(sveList)
+            } else{
+                //sort the SVEs involved in asc order of accession and created date
+                List<SubmittedVariantEntity> sortedList = sveList.stream()
+                        .sorted(Comparator.comparingLong(SubmittedVariantEntity::getAccession)
+                                .thenComparing({ SubmittedVariantEntity sve -> sve.getCreatedDate() }))
+                        .collect(Collectors.toList())
+
+
+                // assume the one in db has the smallest accession
+                Long lowestAcc = sveInDB.getAccession()
+                int startIndex = 0
+
+                SubmittedVariantEntity firstSVEInSorted = sortedList.get(0)
+
+                // update values if the lowest accession is not in db but in sorted List
+                if(firstSVEInSorted.getAccession() < sveInDB.getAccession()){
+                    // we need to process first element separately (it needs to be inserted not discarded or merged)
+                    startIndex = 1
+                    lowestAcc = firstSVEInSorted.getAccession()
+                    // Add the SVE in DB to the list for processing (will be discarded or merged based on the accession)
+                    sortedList.add(sveInDB)
+                    // need to remove this SVE (will be inserted after fixing lower nucleotide)
+                    SVEToRemoveList.add(firstSVEInSorted)
+                    // need to insert the SVE from the sorted list (with lowest accession) into db
+                    SVEToInsertList.add(firstSVEInSorted)
+                }
+
+                for(int i=startIndex; i<sortedList.size(); i++){
+                    SubmittedVariantEntity currSVE = sortedList.get(i)
+                    if(currSVE.getAccession() == lowestAcc){
+                        // if acc is same - discard
+                        SVEToDiscardList.add(currSVE)
+                    }else{
+                        // if acc is different - merge
+                        SVEToMergeMap.computeIfAbsent(lowestAcc, k -> new ArrayList<>()).add(currSVE)
+                    }
+                }
+            }
+        }
+
+        // discard SVEs
+        logger.info("Hash Collision Discard SVE List: " + SVEToDiscardList)
+        discardSVEAndInsertOperations(SVEToDiscardList, ssClass)
+
+        // merge SVEs
+        mergeSVEAndInsertOperations(SVEToMergeMap, ssClass)
+
+        // Insert SVEs
+        logger.info("Hash Collision Insert SVE List: " + SVEToInsertList)
+        insertSVEWithNewHash(SVEToInsertList, ssClass)
+
+        // delete the corrected SVEs
+        logger.info("Hash Collision Remove SVE List: " + SVEToRemoveList)
+        removeImpactedSVE(SVEToRemoveList, ssClass)
```

Comment:
We should count how many are discarded and how many are merged. 
We could get that after from the log I guess.